assets/reports/markdown.template

```diff
@@ -0,0 +1,21 @@
+# osde2e weather report
+
+## Report Information
+This report was generated on {{.ReportDate}}
+
+## Summary
+
+{{range .Jobs}}
+* [{{.Name}}](#{{.Name}}) (Viable: {{.Viable}}){{end}}
+
+{{range .Jobs}}
+## {{.Name}}
+
+Viability: {{.Viable}}
```

Comment:
If we list only jobs that have failures, this becomes redundant and can be removed (because presumably if there are details it's because it's not viable)